tests/test-cases.js

```diff
@@ -1227,10 +1227,9 @@ var CUSTOM_TESTS = [
 				step: function(response, handle) {
 					updates.push(response.data[0]);
 					handle.abort();
-				},
-				complete: function() {
 					callback(updates);
```

Comment:
I've added a test to verify that complete is called after aborting. Unless you see anything else, this should be ready to go.